packages/babel-generator/test/fixtures/sourcemaps/arrow-function-compact/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "compact": true,
```

Comment:
Whoops, need to remove `compact`. I'm trying to see what'll happen if it does insert a newline.